pymeasure/instruments/common_base.py

```diff
@@ -201,13 +243,41 @@ class attributes that are a DynamicProperty. Check also for class variables
                 setattr(self, self.__reserved_prefix + attr, value)
         return special_names
 
+    @staticmethod
+    def get_channels(cls):
+        """Return channels for any BaseChannelCreator channel"""
```

Comment:
According to the docstrings, those static methods should be part of `BaseChannelCreator`, not `MultiChannelCreator`, right?

Reply:
Actually those static methods are part of `Instrument`, which makes sense because we want to return all channels created by both `ChannelCreator` and `MultiChannelCreator` for an instrument. Do you think the docstrings should be clarified?

Reply:
Yes, please, sounds good.

Reply:
Redid those doc strings in latest commits